examples/waspello/src/client/Navbar.jsx

```diff
@@ -1,29 +1,32 @@
-import React from 'react'
+import React from "react";
 
-import logout from '@wasp/auth/logout'
+import logout from "@wasp/auth/logout";
 
-import logo from './waspello-logo-navbar.svg'
-import './Navbar.css'
+import logo from "./waspello-logo-navbar.svg";
+import "./Navbar.css";
 
 const Navbar = ({ user }) => {
-
+  const identifier = user.auth?.identities[0]?.providerUserId;
```

Comment:
I think we should offer some higher level API e.g. a getter or some helper functions.

This approach will also work since right now there is only one identity per user.

Reply:
Are we going to add some higher level API in this PR, I hope? For example we could have "fetchUsername"?
Although I think they should never be doing this, getting first provider user id they can find, it is too wild. I would rather they have User.username which they saved and use that.

We need to give them some clean, nice ways how to use our auth, not let them get into stuff like this.